politeiad/backend/tlogbe/testing.go

```diff
@@ -102,24 +112,89 @@ func newBackendMetadataStream(t *testing.T, id uint64, payload string) backend.M
+			`
+}
+
+func newTestDcrtimeClient(t *testing.T) *dcrtimeClient {
```

Comment:
Is this needed? I don't think it is since the anchor cron job is setup in the tlogBackend initialization function, which means that it is never kicked off in the testTlogBackend init function.

Reply:
you`re right not needed. removed them :+1: